### PR TITLE
Small update to the Payment Handler article

### PR DESCRIPTION
**Pull request by agektmr**
What's changed, or what was fixed?
- An update to https://developers.google.com/web/fundamentals/payments/payment-apps-developer-guide/web-payment-apps#optional_let_chrome_skip-the-sheet
- Updated conditions to skip the Payment Handler sheet

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele @sahel-sh

### CODE REVIEW

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md [hunk 1]
```diff
@@ -421,9 +421,12 @@ Following are the conditions to enable skip-the-sheet:
 
 1. Only one payment method identifier is specified in the `PaymentRequest`
```
Comment (sahel-sh) [inline src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md:422]: This condition is not needed anymore.

Regardless of how many payment method identifiers is specified as long as there is only a single payment handler available with full delegation Chrome would skip to the payment handler if show() is triggered with a user gesture.

Comment (agektmr) [reply src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md]: Fixed.

File: src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md [hunk 2]
```diff
@@ -419,11 +419,13 @@ payment method your app supports.
...
-1. Only one payment method identifier is specified in the `PaymentRequest`
-   constructor. The payment method identifier must be URL-based.
-2. No shipping address or contact info are requested in the `PaymentRequest`
-   constructor.
-3. `show()` is called with a user gesture.
+1. There is only a single payment handler (installed or JIT installable) with
+   full delegation support (i.e. only a single available payment handler can
+   provide all merchant requested information including shipping address and/or
+   contact information whenever needed).
+2. The available payment handler with full delegation should support the
```
Comment (sahel-sh) [inline src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md:426]: the requested URL based -> a requested URL based 
(multiple can be requested)

**Comment by agektmr** [comment]
@petele can you please help us approve this and merge?
Thanks!

**Pull request merged**